locales/bg-BG.json

```diff
@@ -51,7 +51,8 @@
     }
   },
   "date_formats": {
-    "month_year": "%B %Y"
+    "month_year": "%B %Y",
+    "month_day_year": "%B %d, %Y"
```

Comment:
Here you don't have to edit all translation files that have `month_year`. You can only edit it in `en.default.json` and then we will have the translation platform (something we use to generate translations in supported languages based on what is added in the english json files) update all the other files 👍 

Also if we're not using `"month_year"` anymore we could probably remove it.